GeoIP2.UnitTests/project.json

```diff
@@ -3,35 +3,25 @@
   "name": "MaxMind.GeoIP2.UnitTests",
   "description": "Test project to validate MaxMind GeoIP2 Database Reader and Web Service Client",
 
-  "packOptions": {
-    "authors": ["MaxMind, Inc."],
-    "tags": [""],
-    "projectUrl": "https://github.com/maxmind/GeoIP2-dotnet",
-    "licenseUrl": "https://github.com/maxmind/GeoIP2-dotnet/blob/master/LICENSE",
-    "copyright": "Copyright © 2013-2016"
-  },
-
-  "commands": {
-    "test": "MaxMind.GeoIP2.UnitTests"
-  },
-
   "buildOptions": {
-    "emitEntryPoint": true,
     "keyFile": "../MaxMind.snk"
   },
 
   "dependencies": {
     "GeoIP2": {
       "target": "project"
     },
-    "NUnitLite": "3.2.1",
+    "NUnit": "3.4.0",
+    "dotnet-test-nunit": "3.4.0-beta-1",
     "MockHttpSigned": "1.3.0-netstandard-alpha3"
   },
 
+  "testRunner": "nunit",
+
   "frameworks": {
-    "net45": {},
+    "net451": {},
```

Comment:
@oschwald, this change was made because dotnet-test-nunit doesn't support net45. Besides this is unit test project, so I guess this does not matter much? :)